features/FxA-89-devices-view/README.md

```diff
@@ -411,6 +409,17 @@ Acceptance criteria:
       by accessing the devices view on a desktop browser,
       and then go to the sync preferences on the mobile device,
       I immediately see that the device is disconnected.
+* [ ] - Moved from Phase 2: When viewing my device list
```

Comment:
So I suggest we leave it as currently specced in this doc, but add a datadog metrics event for "empty devices view" so that we can see how often people are hitting this in practice - sound ok @vladikoff?